scipy/optimize/optimize.py

```diff
@@ -111,10 +111,10 @@ class OptimizeResult(dict):
 
     Notes
     -----
-    There may be additional attributes not listed above depending of the
-    specific solver. Since this class is essentially a subclass of dict
-    with attribute accessors, one can see which attributes are available
-    using the `keys()` method.
+    `OptimizeResult` may have additional attributes not listed here depending
```

Comment:
```suggestion
    ``OptimizeResult`` may have additional attributes not listed here depending
```

Reply:
If this were just the name of a parameter or something and we wanted it to display like `OptimizeResult`, yes, we'd need double backticks. But in this case single backticks does what we want. It renders like:

![image](https://user-images.githubusercontent.com/6570539/132105705-724f1ea7-347d-4198-9cef-8a522855577d.png)